ui/ducks/send/send.js

```diff
@@ -2670,7 +2670,7 @@ export function updateSendAsset(
 
       if (details.standard === TokenStandard.ERC20) {
         asset.balance =
-          details.balance && typeof details.decimals === 'number'
+          details.balance && details.decimals !== undefined
```

Comment:
I did verify this PR still works for 0 decimal tokens

Reply:
This will also be false when `balance === 0` - should change the `details.balance` to `typeof details.balance`, or is it typed well enough that we know that `details.balance` is never a `number` representation?

Reply:
I dont really know.  Trying to make minimal changes when hotfixing things.   The decimal check is the only bit that changed recently.

Reply:
its true that balance is a string in every place i can reproduce though